.eslintrc.json

```diff
@@ -8,19 +8,17 @@
   "ignorePatterns": ["**/*.mjs", "**/*.js"],
   "extends": [
     "eslint:recommended",
-    "plugin:@typescript-eslint/eslint-recommended",
```

Comment:
The `eslint-recommended` configuration is [automatically included](https://typescript-eslint.io/linting/configs#eslint-recommended) in all of the 'recommended' configs, so there is no need to list it separately.